src/System.Text.Encoding.CodePages/src/System/Text/SBCSCodePageEncoding.cs

```diff
@@ -37,6 +37,27 @@ public SBCSCodePageEncoding(int codePage, int dataCodePage) : base(codePage, dat
         {
         }
 
+        private static unsafe void ZeroMemAligned(byte* buffer, int count)
+        {
+            long* pLong = (long*)buffer;
+            long* pLongEnd = (long*)(buffer + count - sizeof(long));
+
+            while (pLong < pLongEnd)
+            {
+                *pLong = 0;
+                pLong++;
+            }
+
+            byte* pByte = (byte*)pLong;
+            byte* pEnd = buffer + count;
+
+            while (pByte < pEnd)
+            {
+                *pByte = 0;
+                pByte++;
+            }
+        }
+
```

Comment:
I thought having Aligned in the method name was sufficient although will add a comment. Do we have any sandbox to test it on ARM?